.github/workflows/ci.yml

```diff
@@ -3,104 +3,128 @@
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Install Dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r tests/requirements.txt
+        python -m nltk.downloader popular
+
+    - name: Run Unit Tests
+      run: |
+        make test
+
+    - name: Upload Coverage
+      uses: codecov/codecov-action@v1
+      with:
+        env_vars: OS,PYTHON
+        name: codecov-umbrella
+        fail_ci_if_error: false
+
+  # Test Anaconda Python (using conda-forge for dependencies)
   test_conda:
-    name: Conda (${{ matrix.python-version }}, ${{ matrix.os }})
     runs-on: ${{ matrix.os }}
+    name: Conda (${{ matrix.python-version }}, ${{ matrix.os }})
     strategy:
       fail-fast: false
       matrix:
         os: ["ubuntu-latest"]
-        python-version: ["3.7", "3.8"]
+        python-version: ["3.8", "3.9"]
+
     steps:
-      - uses: actions/checkout@v2
-      - uses: conda-incubator/setup-miniconda@v2
+      - name: Checkout Code
+        uses: actions/checkout@v2
+
+      - name: Setup Conda ${{ matrix.python-version }}
+        uses: conda-incubator/setup-miniconda@v2
         with:
           auto-update-conda: true
           python-version: ${{ matrix.python-version }}
           channels: conda-forge,spyder-ide
           channel-priority: flexible
 
-      - name: Conda info
+      - name: Conda Info
         shell: bash -l {0}
         run: conda info
-      - name: Add conda to system path
+
+      - name: Add conda to System $PATH
         shell: bash -l {0}
         run: |
           # $CONDA is an environment variable pointing to the root of the miniconda directory
           echo $CONDA/bin >> $GITHUB_PATH
-      - name: Install dependencies
+
+      - name: Install Dependencies
         shell: bash -l {0}
         run: |
           conda config --set always_yes yes --set changeps1 no
           conda update -n base conda --yes
           conda env create -f tests/requirements.txt -n yellowbrick python=${{ matrix.python-version }}
-      - name: Test with pytest
+          conda activate yellowbrick
+          python -m nltk.downloader popular
+
+      - name: Run Unit Tests
         shell: bash -l {0}
         run: |
           conda activate yellowbrick
-          python -m nltk.downloader popular
           make test
 
-      - name: Upload coverage to Codecov
+      - name: Upload Coverage
         uses: codecov/codecov-action@v1
         with:
           env_vars: OS,PYTHON
           name: codecov-umbrella
           fail_ci_if_error: false
 
-
-  test_pypi:
-    name: PyPi (${{ matrix.python-version }}, ${{ matrix.os }})
-    strategy:
-      fail-fast: false
-      matrix:
-        python-version: [3.7, 3.8]
-        os: [ubuntu-latest, macos-latest, windows-latest]
-
-    runs-on: ${{ matrix.os }}
-
+  # Test building docs using Sphinx since this also runs a lot of Yellowbrick code
+  test_docs:
```

Comment:
Because of the docs build error that prevented the cross-validation visualizer to be shown in the docs I thought it might be good to have a GitHub action for testing the docs build. This test doesn't have to block a PR, but it will give us more visibility into the documentation.

Reply:
Great idea &mdash; the docs behavior is always a little mysterious since it has it's own slightly different requirements.txt; this will be a great feature to add to our CI!